atd-vze/src/queries/gqlAbstract.js

```diff
@@ -318,15 +318,6 @@ gqlAbstractTableAggregateName (
     return this.config["columns"][columnName]["label_" + type] || null;
   }
 
-  /**
-   * Returns the type of the field as implemented in the field configuration
-   * @param {string} columnName - The name of the column
-   * @returns {string|null}
-   */
-  getType(columnName) {
-    return this.config["columns"][columnName]["type"] || null;
-  }
-
```

Comment:
This one is totally my bad, I did not realize I had written this method already. 🤦